File: Witcher-front/src/Components/Armor.js
import React from 'react';

const divStyle = {
    width: '300px',
    border: '2px #000',
    borderRadius: '10px'
};

class Armor extends React.Component {


    render() {
        return(
            <div style={divStyle}>
                {this.props.array.map((elem) => {
                    //if (elem.armorName === this.props.name)
                        return (
                            <div>
                                <table>
                                    <tr>
                                <td>Название - {elem.armorName}</td>
                                <td>Защита - {elem.prot}</td>
                                <td>Цена - {elem.price}</td>
                                    </tr>
                                </table>
                            </div>);
                })}
            </div>
        );
    }

}

export default Armor;

File: Witcher-back/src/main/java/web/witcher/Entities/Armor.java
package web.witcher.Entities;



import lombok.Data;

import javax.persistence.*;

@Entity
@Data
@Table(name = "Armor")
public class Armor {

    @Id
    @Column(name = "armorName")
    private String armorName;

    @Column(name = "price")
    private int price;

    @Column(name = "prot")
    private int prot;

}
